filter/metrics_parser_test.go

```diff
@@ -64,6 +63,7 @@ func TestParseMetric(t *testing.T) {
 			{"One.two.three;four=five 123 1234567890", "One.two.three;four=five", "One.two.three", map[string]string{"four": "five"}, 123, 1234567890},
 			{"One.two.three;four= 123 1234567890", "One.two.three;four=", "One.two.three", map[string]string{"four": ""}, 123, 1234567890},
 			{"One.two.three;four=five;six=seven 123 1234567890", "One.two.three;four=five;six=seven", "One.two.three", map[string]string{"four": "five", "six": "seven"}, 123, 1234567890},
+			{"One.two.three;four=five;six=seven=eight 123 1234567890", "One.two.three;four=five;six=seven=eight", "One.two.three", map[string]string{"four": "five", "six": "seven=eight"}, 123, 1234567890},
```

Comment:
Hi @idoqo I recommend you to add test case with at least two equal signs in value. This case will not work in your implementation.

Example:
```go
{"One.two.three;four=five;six=seven=eight=nine 123 1234567890", "One.two.three;four=five;six=seven=eight=nine", "One.two.three", map[string]string
```

Reply:
Thanks a lot for the catch. I've used a for loop to resolve it and added two new test cases. I don't know if the solution is okay though.